devito/symbolics/inspection.py

```diff
@@ -113,7 +113,10 @@ def estimate_cost(exprs, estimate=False):
                     if all(i.function.is_const for i in terminals):
                         flops += 1
                     else:
-                        flops += trascendentals_cost.get(op.__class__, 1)
+                        try:
```

Comment:
since the cost is 100 for all of them, I actually suggest to finally embrace it -- let's turn `transcental_cost` into a number, ir `trascendentals_cost = 100` and here let's just add `flops += trascendentals_cost`. This will also make it work if a function other than the four present now (sin/cos/log/exp) were used